File: AdrenalinBackend/Domain/Models/New.cs
using Domain.Models.Base;

namespace Domain.Models
{
    public class New : BaseEntity
    {
        public DateOnly Date { get; set; }
        public string? Image { get; set; }
        public string? Title { get; set; }
        public string? Text { get; set; }
    }
}

File: AdrenalinClient/src/components/News/New.js
import React from 'react';
import styles from './new.module.css';

export default function New({ newItem }) {
  return (
    <div className={styles.newWrapper}>
      <div className={styles.date}>
        <label>{newItem.date}</label>
      </div>
      <img
        src={require('../../assets/img/' + newItem.img)}
        alt="new-image"
      ></img>
      <div className={styles.newText}>
        <h4>{newItem.title}</h4>
        <p>{newItem.text}</p>
      </div>
    </div>
  );
}
